infrastructure.md

```diff
@@ -0,0 +1,54 @@
+# CPG Infrastructure
+
+## Dataset
+
+A dataset is one defined collection of resources. It has:
+
+- A GCP project
+- A set of buckets
+- A set of secrets to manage access (access_group_cache, cromwell access keys)
+- A set of allowed_repositories
+
+## Components
+
+- Hail query / batch
+- Analysis-runner
+- Sample-metadata server
+
+
+## Permissions
+
+The [storage policies](storage_policies) document has more information about the different ways we store data in the CPG, but broadly, each dataset has three levels:
+
+- `test`: can run any commit on the allowed repositories, on the `dataset-test*` buckets
+- `standard`: commit MUST be on the main branch, allowed READ / WRITE access to `dataset-main*` buckets
+- `full`: commit MUST be on the branch branch, allowed all operations on `dataset-main*` and `dataset-test` buckets
+
+## Example: seqr loading pipeline
+
+The seqr loading pipeline is good example, as it uses almost every single piece of architecture that the CPG offers.
+
+This relies on:
+
+- samples being loaded into a `dataset-main*` bucket
+- sample metadata being loaded into the sample-metadata system
+    - Including pedigree data, individual level metadata (eg: HPO terms)
```

Comment:
This is kind of vague -- can you outline here what the soonish picture of seqr wizard + sample metadata API interactions look like? For someone not familiar with our process, it's unclear what "loaded" means in this context.